orchagent/main.cpp

```diff
@@ -505,6 +505,10 @@ int main(int argc, char **argv)
     attr.value.ptr = (void *)on_switch_shutdown_request;
     attrs.push_back(attr);
 
+    attr.id = SAI_SWITCH_ATTR_PORT_HOST_TX_READY_NOTIFY;
```

Comment:
Does it handle if a platform does not support this notification?

Reply:
Sure, this line purpose is only to have the callback function for the notification. 
In PortsOrch we have 2 queries to check whether this functionality is supported, and if not, the notification will not be registered and this line will not be relevant.